back_end/RayTracer.userprefs

```diff
@@ -1,19 +1,27 @@
 ﻿<Properties StartupConfiguration="{E7016944-4EBD-4FB0-9113-DA838BAA1D98}|">
-  <MonoDevelop.Ide.Workbench ActiveDocument="RayTracer/Models/Geometric/Sphere.cs">
+  <MonoDevelop.Ide.Workbench ActiveDocument="RayTracer/Controllers/IndexController.cs">
     <Files>
-      <File FileName="RayTracer/Models/Cameras/Camera.cs" Line="21" Column="25" />
+      <File FileName="RayTracer/Models/Cameras/Camera.cs" Line="14" Column="31" />
       <File FileName="RayTracer/Models/Geometric/Sphere.cs" Line="14" Column="20" />
       <File FileName="RayTracer/Models/Util/Config.cs" Line="41" Column="10" />
       <File FileName="RayTracer/Models/Util/HitInfo.cs" Line="19" Column="38" />
-      <File FileName="RayTracer/Controllers/RenderController.cs" Line="1" Column="1" />
+      <File FileName="RayTracer/Controllers/RenderController.cs" Line="14" Column="59" />
+      <File FileName="RayTracer/App_Start/WebApiConfig.cs" Line="13" Column="39" />
+      <File FileName="RayTracer/Web.config" Line="18" Column="9" />
+      <File FileName="RayTracer/Models/Json/JsonObject.cs" Line="11" Column="32" />
+      <File FileName="RayTracer/Global.asax.cs" Line="13" Column="42" />
+      <File FileName="RayTracer/Controllers/IndexController.cs" Line="38" Column="9" />
+      <File FileName="RayTracer/App_Start/RouteConfig.cs" Line="1" Column="1" />
+      <File FileName="RayTracer/packages.config" Line="3" Column="32" />
+      <File FileName="RayTracer/Views/Web.config" Line="4" Column="82" />
     </Files>
   </MonoDevelop.Ide.Workbench>
   <MonoDevelop.Ide.Workspace ActiveConfiguration="Debug" />
   <MonoDevelop.Ide.DebuggingService.Breakpoints>
     <BreakpointStore>
-      <Breakpoint file="/Users/othmanalkhamra/Documents/GitHub/raytracer/back_end/RayTracer/Models/Json/JsonObject.cs" relfile="RayTracer/Models/Json/JsonObject.cs" line="32" column="1" />
```

Comment:
Just noticed this, probably shouldn't have that in git

Reply:
Deleted :)